### PR TITLE
Document that we now release to CAAS as well as Terra [BA-5902]

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: processes/release_processes/README.MD [hunk 1]
```diff
@@ -30,13 +34,17 @@ for any performance changes since the previous release.
...
 **Note:** Full details on how to complete each of these steps is found in the [Quick CAAS Deployment Guide](https://docs.google.com/document/d/1s0YC-oohJ7o-OGcgnH_-YBtIEKmLIPTRpG36yvWxUpE) 
 
 ![caas-prod](caas-prod.dot.png)
 
-## How to Generate and Publish Swagger Client Library
+## Bonus Processes
+
+The swagger client library is not part of our core publish/release process but can be performed from time to time, as required.
+
+### How to Generate and Publish Swagger Client Library
```
Comment (salonishah11) [inline processes/release_processes/README.MD]: Can you explain what is Swagger Client Library? I am not sure how it is related to Cromwell release

Comment (cjllanwarne) [reply processes/release_processes/README.MD]: This was added by @kcibul I think - and I'm just shuffling it around. 

I think it's a way of creating swagger client libraries from the yaml definitions but it's still not part of the main release process (hopefully the new headings in this doc make that clear?)

Comment (salonishah11) [reply processes/release_processes/README.MD]: Ah I see. Thank you! :)

File: processes/release_processes/README.MD [hunk 2]
```diff
@@ -5,11 +5,15 @@
 Have a better idea about how the deployment processes should work? 
 See our "updating the process" [process](../README.MD)!
 
-## How to Publish Cromwell
+## Core Process: How to Publish and Release Cromwell
 
-![release-cromwell-version](release-cromwell-version.dot.png) 
+If you're the lucky "Release Rotation Engineer" this time, you should do the following three things:
```
Comment (aednichols) [inline processes/release_processes/README.MD]: Strictly personal preference, but I prefer my mission-critical checklists to be 100% factual and humorless

**Pull request merged**